evalai/utils/text_styling.py

```diff
@@ -0,0 +1,27 @@
+
+from click import echo, style
+
+
+def print_message(message, error=False):
+
+    style_message = ""
+
+    index = 0
+
+    for i in list(message.split("`")):
+
+        if index % 2:
+            style_message += style(i, fg="cyan")
+
+        elif error:
+            style_message += style(i, fg="red", bold=True)
+
+        else:
+            style_message += style(i, fg="green")
```

Comment:
You used the `error` flag here to either print the error message or the success message. But there could be times when we just want to print the info with white bold text or just plain white text. Can you add that functionality here somehow? Feel free to use helper methods as mentioned in #234 

Small query: `bold=True`?

Reply:
Ok, I will include it in following PR.

Reply:
I just followed the codes of the other files. most of the error messages where bold in those files so I kept it bold.

Reply:
I will add an argument to the functions for bold text in the following PR